schema.c

```diff
@@ -127,6 +127,16 @@ sch_last_errmsg (void)
     return tl_errmsg;
 }
 
+static gpointer
+copy_node_data (gconstpointer src, gpointer dummy)
+{
+    char *data = NULL;
+
+    if (src)
+        data = g_strdup (src);
```

Comment:
FYI - g_strdup works with NULL pointers too.

Reply:
OK - I will change the code